pkg/build/devcontainer.go

```diff
@@ -36,12 +36,12 @@ type BuildOutcome struct {
 
 type DevcontainerBuilder struct {
 	*Builder
-	buildImageName    string
+	buildImage        string
```

Comment:
I suggest considering removing this property.

I believe the image and user should be returned from the `buildDevcontainer` function.

If there is a strong reason for keeping this, please elaborate and we can keep it.